samples/powerpoint/basics/insert-image.yaml

```diff
@@ -0,0 +1,62 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: insert-image
+description: Inserts an image to the current slide.
+script:
+  content: |
+    $('#insert').click(run);
+
+    async function run() {
+        var base64EncodedImageStr = getImageAsBase64String();
+
+        await Office.context.document.setSelectedDataAsync(base64EncodedImageStr, {
+            coercionType: Office.CoercionType.Image,
+            imageLeft: 50,
+            imageTop: 50,
+            imageWidth: 400
+        },
+            function (asyncResult) {
```

Comment:
That would make the code less readable. Also, this method doesn't return a value. If this was a production app, I'd encapsulate the error handling in a named function and call the function here. But since its a playground designed to teach the Office APIs, I think an anonymous method is OK. But no additional purpose is served by making it a lambda function. It is _not_ a best practice to "use a lambda function whenever you can use one."